### ISSUE TITLE
Exception capture not working with clojure-maven-plugin

### ISSUE THREAD

**Issue by sanel** [sanel/monroe#20]
Using [clojure-maven-plugin](https://github.com/talios/clojure-maven-plugin), in case of exception, monroe will sometimes ends up in loop with this output:

```
demo=> ClassNotFoundException clojure.repl  java.net.URLClassLoader.findClass (URLClassLoader.java:381)
demo=> ClassNotFoundException clojure.repl  java.net.URLClassLoader.findClass (URLClassLoader.java:381)
demo=> ClassNotFoundException clojure.repl  java.net.URLClassLoader.findClass (URLClassLoader.java:381)
demo=> ClassNotFoundException clojure.repl  java.net.URLClassLoader.findClass (URLClassLoader.java:381)
demo=> ClassNotFoundException clojure.repl  java.net.URLClassLoader.findClass (URLClassLoader.java:381)
demo=> ClassNotFoundException clojure.repl  java.net.URLClassLoader.findClass (URLClassLoader.java:381)
demo=> ClassNotFoundException clojure.repl  java.net.URLClassLoader.findClass (URLClassLoader.java:381)
demo=> ClassNotFoundException clojure.repl  java.net.URLClassLoader.findClass (URLClassLoader.java:381)
demo=> ClassNotFoundException clojure.repl  java.net.URLClassLoader.findClass (URLClassLoader.java:381)
demo=> ClassNotFoundException clojure.repl  java.net.URLClassLoader.findClass (URLClassLoader.java:381)
demo=> ClassNotFoundException clojure.repl  java.net.URLClassLoader.findClass (URLClassLoader.java:381)
demo=> ClassNotFoundException clojure.repl  java.net.URLClassLoader.findClass (URLClassLoader.java:381)
demo=> ClassNotFoundException clojure.repl  java.net.URLClassLoader.findClass (URLClassLoader.java:381)
demo=> ClassNotFoundException clojure.repl  java.net.URLClassLoader.findClass (URLClassLoader.java:381)
demo=> ClassNotFoundException clojure.repl  java.net.URLClassLoader.findClass (URLClassLoader.java:381)
d
```

Reason is that `clojure-maven-plugin` is not loading `clojure.repl` namespace.

**Comment by technomancy** [comment]
It happens also with `lein repl :headless` and most other situations in which nRepl is embedded in an application.
-- 
Sent from my Android device with K-9 Mail. Please excuse my brevity.

**Comment by sanel** [comment]
Didn't know about this. Thanks for letting me know.

**Comment by technomancy** [comment]
Added a fix to this to #19.
